packages/notebook/src/default-toolbar.tsx

```diff
@@ -150,6 +153,28 @@ export namespace ToolbarItems {
       tooltip: 'Run the selected cells and advance'
     });
   }
+  /**
+   * Create a restart run all toolbar item
+   */
+  export function createRestartRunAllButton(
+    panel: NotebookPanel,
+    dialogs?: ISessionContext.IDialogs
+  ): Widget {
+    return new ToolbarButton({
+      icon: fastForwardIcon,
+      onClick: () => {
+        void (dialogs ?? sessionContextDialogs)
+          .restart(panel.sessionContext)
+          .then(restarted => {
+            if (restarted) {
+              void NotebookActions.runAll(panel.content, panel.sessionContext);
+            }
+            return restarted;
+          });
+      },
+      tooltip: 'Restart the kernel, then re-run the whole notebook'
```

Comment:
```suggestion
      tooltip: 'Restart Kernel and Run All Cells…'
```
Can we use the language from the command in JupyterLab for consistency with the lab interface?

Reply:
Hmm, never mind on this one. This text is more consistent with the other tooltips.